components/Builder.tsx

```diff
@@ -63,7 +63,7 @@ export default function Builder({ data, formation: _formation }: Props) {
   const [layout, setLayout] = useQueryState<number>("map", {
     parse: (query: string): number => parseInt(query),
     serialize: (layout: number) => layout.toString(),
-    defaultValue: _formation ? _formation.layout : 0,
+    defaultValue: _formation ? Math.trunc(_formation.layout) : 0,
```

Comment:
curious what this fixed?

Reply:
this fixed a situation where some of our older formations had values like `0.0` for their `layout`.

by doing `Math.trunc` we ensure that the values are coerced into integers for use later on in the `layouts` lookup.

in the future this process will get phased out, but for now this ensures we don't hit any weird bugs.